python/generate/templates/api.mustache

```diff
@@ -18,7 +18,7 @@ from {{packageName}}.exceptions import (
 
 {{#operations}}
 class {{classname}}(object):
-    """NOTE: This class is auto generated by OpenAPI Generator
```

Comment:
Where are the Python default templates?

Reply:
Have added those now. I also made it use the `python-legacy` type in Open API because using `python` was breaking all the things.